teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>be meaningful metric &gt; be <strong>a </strong>meaningful metric</p><p>dataset dependent &gt; dataset-dependent (?)</p><p>prediction errors respect to the line of best fit &gt; prediction errors <strong>with</strong> (?) respect to the line of best fit</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>